collective/contentrules/mailtogroup/actions/mail.py

```diff
@@ -96,16 +103,75 @@ def __init__(self, context, element, event):
         self.context = context
```

Comment:
I believe it's needed, because the MailActionExecutor is a multi-adapter from a context, an action and an event.